### PR TITLE
Less tidy up

### PR DESCRIPTION
**Pull request by guywillis**
Related issue: https://github.com/adaptlearning/adapt_framework/issues/2891

### CODE REVIEW

File: less/narrative.less [hunk 1]
```diff
@@ -178,54 +171,46 @@
...
+    display: none;
   }
 
-  &__text-controls {
-    .narrative__content {
-      position: relative;
-      padding-top: @icon-size + (@icon-padding * 2);
+  &__text-controls &__content {
+    position: relative;
+    padding-top: @icon-size + (@icon-padding * 2) + @item-padding;
```
Comment (cahirodoherty-learningpool) [inline less/narrative.less:184]: Was adding @item-padding here intentional?

Comment (guywillis) [reply less/narrative.less:184]: It was though I should have commented on it. During development I noticed that there was no gap between the text controls and narrative content and so introduced the additional padding to give the elements some room to breath. 

Would you like me to revert the above and raise a new issue?

Comment (cahirodoherty-learningpool) [reply less/narrative.less:184]: That's fine then. I'm happy to keep it in

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by kirsty-hames — APPROVED** [review]
👀

**Review by kineojen — APPROVED** [review]

**Pull request merged**